### PR TITLE
Add numbered point about merging strategy

### PR DESCRIPTION
**Pull request by henning-roos**
### Description of the Change
Add numbered point about merging strategy.

### Benefits
All new repositories will adhere to the CONTRIBUTING.md document.

### Possible Drawbacks
None

### Sign-off
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Henning Roos henning.roos@ericsson.com

### CODE REVIEW

**Comment by henning-roos** [comment]
Note that this is not needed if  #7 is solved.

File: repo-checklist.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ After copying the contents of this repo into the new repo-to-be, follow the belo
    1. Replace the logo and the alt text of the logo image element. You can find logo template here:  https://github.com/eiffel-community/community/blob/master/Resources/eiffel-community-logos.xcf
    1. Change the first heading and the description of the repository.
    1. Keep the rest of the README.md intact (license notice, links to contribution guidelines et cetera).
```
Comment (e-pettersson-ericsson) [inline repo-checklist.md:24]: There is a typo in "Allow"

**Review by e-pettersson-ericsson — APPROVED** [review]
Looks good, but I think I see the appeal even more with #7 and making this a proper template. I assume such settings are copied also.

**Review by e-backmark-ericsson — APPROVED** [review]
LGTM

**Pull request merged**